addons/dragging/functions/fnc_dropObject_carry.sqf

```diff
@@ -46,6 +46,7 @@ if (_tryLoad && {!(_target isKindOf "CAManBase")} && {["ace_cargo"] call EFUNC(c
 } else {
     // Release object
     detach _target;
+    [QGVAR(objectDroppedCarry), [_unit, _target]] call CBA_fnc_localEvent;
```

Comment:
I would place this inside the main scope and pass `_loadCargo`, so that 3rd party mods can intervene even if the item will be loaded into cargo.